kernel-crawler/rhel-subscription-crawler/crawl.sh

```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/env bash
 set -eou pipefail
 
-rhel_package_lists=/tmp/rhel_package_lists.txt
-ls /kernel-package-lists/rhel[0-9]*.txt > "$rhel_package_lists"
+#rhel_package_lists=/tmp/rhel_package_lists.txt
+#ls /kernel-package-lists/rhel[0-9]*.txt > "$rhel_package_lists"
```

Comment:
What was the reason for commenting this out? This was created so that RHSM API crawler would not duplicate packages already crawled by other crawlers. Are you turning off those other crawlers? If so something should be done about this if those other crawlers are turned back on.

Reply:
Yes, since we can't use subscription manager, everything will go through the rhsm api.